res/database/Default/user/scr0llbaer_constr_roles.xml

```diff
@@ -0,0 +1,343 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tvtdb>
+	<version value="3" comment="Role entries under construction. Incomplete, not validated, and not tested. For build-stage review, feedback and coordination with other contributors only. Contents will be moved to appropriate places, and this file will be deleted in the last step of the PR." />
+
+	<programmeroles>
+
+		<!-- Roles based on common folklore, fairytales, mythology etc characters, most also in the language files! -->
+
+		<programmerole guid="a00ea58a-4a9f-466e-9fe5-cf8b53730b60" first_name="Snow White" last_name="" first_name_original="Snow White" last_name_original="" gender="2" comment="https://en.wikipedia.org/wiki/Snow_White" />
```

Comment:
Why not use readable ids for all roles. This would make the templates more readable as well.

Reply:
Same reason why databases usually don't use readable IDs in general I guess. Because readable real-world things can change.

Reply:
For persons/roles in TVTower, I don't see this. Fixing typos - OK, but would you really change the role name from James Bond to Mickey Mouse?